packages/trpc/server/routers/loggedInViewer/outOfOffice.handler.ts

```diff
@@ -32,6 +32,8 @@ export const outOfOfficeCreateOrUpdate = async ({ ctx, input }: TBookingRedirect
 
   const inputStartTime = dayjs(startDate).startOf("day");
   const inputEndTime = dayjs(endDate).endOf("day");
+  const startDateUtc = dayjs.utc(startDate).add(input.offset, "minute");
+  const endDateUtc = dayjs.utc(endDate).add(input.offset, "minute");
```

Comment:
Good use of startDateUtc and endDateUtc for consistent date handling. Should help avoid time zone issues